dev/knn.py

```
@@ -0,0 +1,197 @@
+    grid = []
+    for lat in lat_steps:
+        for lon in lon_steps:
+            grid.append((lat, lon))
+    return grid
+
+grid_points = make_grid(boundary, step_m=100)
+print(f"Grid points count: {len(grid_points)}")
+# print(grid_points)  # Uncomment to see all grid coordinates
+
+import requests
+import requests
```

Ruff (0.15.2)

[error] Redefinition of unused `requests` from line 75: `requests` redefined here

Remove definition: `requests`

(F811)